README.md

```diff
@@ -1,2 +1,76 @@
-# ts_m2fpga
-M2 Controller FPGA code
+# M2 Cell Controller FPGA Code
```

Comment:
Give the full name of "FPGA" before the abbreviation.

Reply:
Done